portfolio/src/main/java/com/google/sps/servlets/BackgroundChange.java

```diff
@@ -0,0 +1,35 @@
+package com.google.sps.servlets;
+import java.util.ArrayList;
+import java.util.List;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns imageURL to change the background of index.html*/
+@WebServlet("/background-change")
+public class BackgroundChange extends HttpServlet {
+  private List<String> images;
+  private int prevIndex = -1;
+  @Override
+  public void init() {
+      images = new ArrayList<>();
```

Comment:
Every time you add a new image for the random selector, you'll have to remember to change this list. If you instead use a mechanism like getting the filenames from this directory and iterating over them, simply adding an image to the "image" directory or a new subdirectory for backgrounds would be sufficient. Not having to make a change in two places makes your code much easier to maintain.

Reply:
That is a great suggestion! I'll change that.